### ISSUE TITLE
Rounded corner box characters don't seem to be rendered correctly

### ISSUE THREAD

**Issue by npezza93** [xtermjs/xterm.js#3894]
Rounded corner characters are rendered at kind of an angle and differently than the default macOS terminal or rendered in the browser. I have tried Fira code, jet brains mono, and sf mono fonts and all render the same way. I have also tried all variations of turning on and off canvas, webgl, and ligatures and all render the same way. 

## Details
- Browser and browser version: electron 19
- OS version: macOS Monterey 
- xterm.js version: 4.18 and 4.19

### Steps to reproduce

1. echo "╭─╮\n\r│ │\n\r╰─╯\n\r"

xterm:
<img width="769" alt="image" src="https://user-images.githubusercontent.com/5059927/178127547-bbe7272a-d028-45d5-b4bd-9d58ea2cab05.png">

default macOS terminal:
<img width="516" alt="image" src="https://user-images.githubusercontent.com/5059927/178127574-2dcb1293-2bb3-463a-8d70-450b503c4706.png">

**Comment by Tyriar** [comment]
I agree it would look better like you suggest. You can change to the font rendering by setting `customGlyphs` to false, they won't be properly aligned then though.
